### ISSUE TITLE
pull-push triplet loss损失函数不一致

### ISSUE THREAD

**Issue by jige2314** [Ferenas/DAPT#5]
尊敬的作者您好：
我发现您论文中的pull-push triplet loss损失函数 与 `实际上您代码中的损失函数不一致，这是为什么呢？您在论文中是L1损失：
`$L_v = \sum_{i=1}^{b} \max( \| \tilde{Z}_I^i - \tilde{Z}_f^i \|_1 - \| \tilde{Z}_I^i - \tilde{Z}_b^i \|_1 + \alpha, 0 )$`，
但是代码中使用的确实L2损失：
loss_re1 = F.triplet_margin_loss(image_features,image_features_fg.detach(),image_features_bg.detach(),margin=1.5)，代码在
DAPT-main/trainers/maple.py中；
这个是为什么呢？
